### PR TITLE
fix(message error): convert time in message error

### PR DESCRIPTION
**Pull request by cornhubius**
**What**

- fixed error message in `delete_day` and `delete_interval`, now the time is converted to the user's timezone
- added deletion message in `delete_all`

related #18

### CODE REVIEW

**Review by leopardilla — APPROVED** [review]

File: backend/cli/delete_day.py [hunk 1]
```diff
@@ -14,4 +14,5 @@ def delete_day(params):
         cur.execute("DELETE" + QUERY + "is null", [params_start, params_end])
         cur.execute("SELECT start_interval" + QUERY + "NOT null", [params_start, params_end])
         for result in cur:
-            print("""Can't delete booked interval: {}""".format(result))
+            result = result[0]
```
Comment (mentortechabc) [inline backend/cli/delete_day.py:17]: You may notice that you had to make the same changes in two similar functions. That may be a sign that the code is repetitive and refactoring may be a good option.

**Review by mentortechabc — DISMISSED** [review]

**Comment by mentortechabc** [comment]
"related #86" - we don't have a ticket 86. Do I miss something?

@cornhubius could you please resolve the conflicts?

**Comment by cornhubius** [comment]
> "related #86" - we don't have a ticket 86. Do I miss something?
> 
> @cornhubius could you please resolve the conflicts?
мой косяк, эта задача была в старом репозитории

**Comment by mentortechabc** [comment]
blocked by failing automated tests

File: backend/cli/db/convert_time.py [hunk 2]
```diff
@@ -48,3 +48,9 @@ def collapse_intervals(lst):
     for single_dt in single_lst:
         single_interval = ("""{} - {}""".format(single_dt, single_dt + timedelta(minutes=15)))
         yield single_interval
+
+
+def tuple_to_list(cur):
```
Comment (mentortechabc) [inline backend/cli/db/convert_time.py:53]: The name of the function doesn't help the reader to understand what it does. Can you find a better name?

**Review by mentortechabc — CHANGES_REQUESTED** [review]

File: backend/cli/db/convert_time.py (hunk 2 above)
Comment (cornhubius) [reply backend/cli/db/convert_time.py:53]: Maybe this name will do?😊 -> `convert_tuple_to_list`

Comment (mentortechabc) [reply backend/cli/db/convert_time.py:53]: well, where is the `list` you are talking about?

Comment (cornhubius) [reply backend/cli/db/convert_time.py:53]: Раньше информация выводилась в формате `('2021-05-03 05:15:00',)` и время было не сконвертировано. Здесь на вход подается кортеж из которого берется первое значение и выводит сконвертированное значение в формате `2021-05-03 08:15:00`. Может `print_convert_time`?

**Review by mentortechabc — APPROVED** [review]

**Review by DramaTourG — APPROVED** [review]

**Review by leopardilla — APPROVED** [review]

**Pull request merged**